assets/badge.js

```diff
@@ -111,7 +111,7 @@
     div.style.lineHeight = '0'
     div.style.backgroundColor = '#fafafa'
     div.style.width = '25em'
-    div.style.height = '15.5em'
+    div.style.height = '30em'
```

Comment:
The badge changes are unrelated

Reply:
The badge popup size had to be increased to make room for another checkbox